### PR TITLE
[12.0][MIG] maintenance_equipment_contract

### PR DESCRIPTION
**Pull request by jarroyomorales**
Migration to 12.0 of maintenance_equipment_contract.
`'account.analytic.account'` became `contract.contract` so a few changes were needed.

@etobella  @dalonsod

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM 👍

File: maintenance_equipment_contract/views/maintenance_equipment.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    <record model="ir.ui.view" id="hr_equipment_view_form">
+        <field name="name">maintenance.equipment.form (in maintenance_equipment_contract)</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form"/>
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <field name="contract_ids" invisible="1"/>
+                <button name="action_view_contracts" class="oe_stat_button"
+                        icon="fa-usd" type="object" attrs="{'invisible': [('contract_ids', '=', [])]}">
```
Comment (dalonsod) [inline maintenance_equipment_contract/views/maintenance_equipment.xml]: With this invisible condition, you cannot create a contract from equipment form, when the equipment hasn't any contract yet, you need to yo to Invoicing menu to achieve it. But, if it has got one or more contracts, creating a contract ithin an equipment is already enabled. Maybe this button should be always visible. What do you think?

**Review by dalonsod — CHANGES_REQUESTED** [review]
Tested on runbot, only a functional doubt and pending squash commits

**Comment by jarroyomorales** [comment]
@dalonsod Yes, I agree with you, just made it always visible to be more consistent. Thank you for the review!

**Review by dalonsod — APPROVED** [review]
LGTM 👍 
Don't forget to squash commits

**Pull request merged**